airflow/operators/docker_operator.py

```diff
@@ -228,7 +228,10 @@ def execute(self, context):
             self.cli.start(self.container['Id'])
 
             line = ''
-            for line in self.cli.logs(container=self.container['Id'], stream=True):
+            for line in self.cli.attach(container=self.container['Id'],
+                                        stdout=True,
+                                        stderr=True,
+                                        stream=True):
```

Comment:
You have to create JIRA ticket,because you change code

Reply:
@ashwiniadiga You CI test failed due to you don not change unit test code, please change `tests/operators/test_docker_operator.py` to pass CI test

Reply:
https://issues.apache.org/jira/browse/AIRFLOW-3746, 
I fixed the unit tests

Reply:
@zhongjiajie I have fixed the  all failing tests. Can you please review

Reply:
@ashwiniadiga LGTM